app/graphql/types/customers/shipping_address_input.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Types
+  module Customers
+    class ShippingAddressInput < BaseInputObject
```

Comment:
Same here, what about an `AddressInput` type?

Reply:
TBH, I haven't found approach like gql interface for the input type. Did you have anything specific in mind?

Reply:
My point was to ask, do we need a specific `ShippingAddress`? It could just be an `Address` that could be reused in multiple places.

Reply:
Updated